README.md

```diff
@@ -111,3 +111,17 @@ Special thanks to the following people who helped beta test this GitHub Action a
 - [uakfdotb](https://github.com/uakfdotb)
 - [hexecute](https://github.com/hexecute)
 - [JonathanZhu11](https://github.com/JonathanZhu11)
+
+## Updated by [Aly Ghallab](https://github.com/AlyHG) to Handle Locked LeetCode Premium Problems and Improve Sync Reliability - June 19, 2024
```

Comment:
Would it be okay if I add this info in the release notes section when creating a new release? e.g. similar to https://github.com/joshcai/leetcode-sync/releases/tag/v1.6